exchanges/subscription/store_test.go

```diff
@@ -43,11 +45,20 @@ func TestNewStoreFromList(t *testing.T) {
 
 // TestAdd exercises Add and add methods
 func TestAdd(t *testing.T) {
-	assert.ErrorIs(t, (*Store)(nil).Add(&Subscription{}), common.ErrNilPointer, "Should error nil pointer correctly")
-	assert.ErrorIs(t, (&Store{}).Add(nil), common.ErrNilPointer, "Should error nil pointer correctly")
-	assert.ErrorIs(t, (&Store{}).Add(&Subscription{}), common.ErrNilPointer, "Should error nil pointer correctly")
+	t.Parallel()
+	err := (*Store)(nil).Add(&Subscription{})
+	assert.ErrorIs(t, err, common.ErrNilPointer, "Should error nil pointer correctly")
+	assert.ErrorContains(t, err, "called on nil Store", "Should error correctly")
+
+	err = new(Store).Add(nil)
+	assert.ErrorIs(t, err, common.ErrNilPointer, "Should error nil pointer correctly")
+	assert.ErrorContains(t, err, "called on an Uninitialised Store", "Should error correctly")
```

Comment:
```suggestion
	assert.ErrorContains(t, err, "called on an uninitialised Store", "Should error correctly")
```

Reply:
This implies the error itself is wrong.

Fixed 4d4d49e